Buid failure: "Requires at least Gradle 6.0, but was run with 5.0"

Running the build as instructed but slightly adapting the `build_ghidra.sh` to include the `-b patch` flag to change the branch

```sh
if [ ! -d "ghidra" ]; then
    git clone -b patch https://github.com/NationalSecurityAgency/ghidra
fi
```

results in

```
Copying vsOlder_x86.fidb to /files/ghidra/dependencies/fidb

FAILURE: Build failed with an exception.

* Where:
Build file '/files/ghidra/build.gradle' line: 29

* What went wrong:
A problem occurred evaluating root project 'ghidra'.
> Requires at least Gradle 6.0, but was run with 5.0

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7m 1s
```

Though I think it would have also failed if I gave it the regular branch.

--- a/docker-tpl/Dockerfile
+++ b/docker-tpl/Dockerfile
@@ -33,10 +33,10 @@ USER dockerbot
 # Build tools are part of the builder image; project code is mounted
 RUN mkdir -p $HOME/.gradle/init.d/
 
-RUN wget https://services.gradle.org/distributions/gradle-5.0-bin.zip -P /tmp
+RUN wget https://services.gradle.org/distributions/gradle-6.0-bin.zip -P /tmp
 RUN unzip -d $HOME/gradle /tmp/gradle-*.zip
 
-ENV GRADLE_HOME=/home/dockerbot/gradle/gradle-5.0
+ENV GRADLE_HOME=/home/dockerbot/gradle/gradle-6.0
 ENV PATH=$GRADLE_HOME/bin:$PATH
 
 RUN echo "\

